packages/components/src/components/LinkingProvider.tsx

```diff
@@ -71,14 +67,6 @@ export interface LinkLibrary {
   PackageDependants: (
     props: AnyProps & { community: string; package: string }
   ) => RE | null;
-  /** Link for downloading the package */
-  PackageDownload: (
-    props: AnyProps & { namespace: string; package: string; version: string }
-  ) => RE | null;
-  /** Link for installing the package via mod manager */
-  PackageInstall: (
```

Comment:
Is this removal a good idea? How do we implement the install button in the mod manager, where it probably shouldn't open a link but instead run the actual action?

Reply:
Let's figure this out in a separate PR